src/panels/KaitoManagePanel.ts

```diff
@@ -0,0 +1,206 @@
+import { BasePanel, PanelDataProvider } from "./BasePanel";
+import * as vscode from "vscode";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { MessageHandler, MessageSink } from "../webview-contract/messaging";
+import { ToVsCodeMsgDef, ToWebViewMsgDef, ModelState } from "../webview-contract/webviewDefinitions/kaitoManage";
+import { InitialState } from "../webview-contract/webviewDefinitions/kaitoManage";
+import { TelemetryDefinition } from "../webview-contract/webviewTypes";
+import { invokeKubectlCommand } from "../commands/utils/kubectl";
+import { failed } from "../commands/utils/errorable";
+import { longRunning } from "../commands/utils/host";
+import { getConditions, convertAgeToMinutes } from "./utilities/KaitoHelpers";
+import { join } from "path";
+import { writeFileSync } from "fs";
+import { tmpdir } from "os";
+
+export class KaitoManagePanel extends BasePanel<"kaitoManage"> {
+    constructor(extensionUri: vscode.Uri) {
+        super(extensionUri, "kaitoManage", {
+            monitorUpdate: null,
+        });
+    }
+}
+
+export class KaitoManagePanelDataProvider implements PanelDataProvider<"kaitoManage"> {
+    public constructor(
+        readonly clusterName: string,
+        readonly subscriptionId: string,
+        readonly resourceGroupName: string,
+        readonly armId: string,
+        readonly kubectl: k8s.APIAvailable<k8s.KubectlV1>,
+        readonly kubeConfigFilePath: string,
+        readonly models: ModelState[],
+    ) {
+        this.clusterName = clusterName;
+        this.subscriptionId = subscriptionId;
+        this.resourceGroupName = resourceGroupName;
+        this.armId = armId;
+        this.kubectl = kubectl;
+        this.kubeConfigFilePath = kubeConfigFilePath;
+        this.models = models;
+    }
+
+    getTitle(): string {
+        return `Manage Kaito Models`;
+    }
+    getInitialState(): InitialState {
+        return {
+            clusterName: this.clusterName,
+            models: this.models,
+        };
+    }
+    getTelemetryDefinition(): TelemetryDefinition<"kaitoManage"> {
+        return {
+            monitorUpdateRequest: false,
```

Comment:
if these variables are false we won't have telemetry for these, is that intentional?

Reply:
Re-enabling them in these last commits, just wanted to make sure they were off for the testing phase